README.md

```diff
@@ -93,39 +99,43 @@ npx hardhat test --network example --grep 'AxelarGateway'
+
 ```javascript
 await sourceChainGateway
-         .execute(
-               await getSignedWeightedExecuteInput(await getTokenDeployData(false), [operatorWallet], [1], 1, [operatorWallet]),
-               getGasOptions(),
-         )
-         .then((tx) => tx.wait(network.config.confirmations));
+    .execute(
+        await getSignedWeightedExecuteInput(await getTokenDeployData(false), [operatorWallet], [1], 1, [operatorWallet]),
+        getGasOptions()
+    )
+    .then((tx) => tx.wait(network.config.confirmations));
 ```
 
-- Using the most up to date and fast rpc can help in tests execution runtime. Make sure the rate limit for the rpc is not exceeded.
+-   Using the most up to date and fast rpc can help in tests execution runtime. Make sure the rate limit for the rpc is not exceeded.
+
+-   Make sure that the account being used to broadcast transactions has enough native balance. The maximum `gasLimit` for a chain should be fetched from an explorer and set it in config file. You may also need to update the `confirmations` required for a transaction to be successfully included in a block in the config [here](https://github.com/axelarnetwork/axelar-contract-deployments/tree/main/axelar-chains-config/info) depending on the network.
 
-- Make sure that the account being used to broadcast transactions has enough native balance. The maximum `gasLimit` for a chain should be fetched from an explorer and set it in config file. You may also need to update the `confirmations` required for a transaction to be successfully included in a block in the config [here](https://github.com/axelarnetwork/axelar-contract-deployments/tree/main/axelar-chains-config/info) depending on the network.
+-   Note that certain tests can require upto 3 accounts.
 
-- Note that certain tests can require upto 3 accounts.
+-   Transactions can fail if previous transactions are not mined and picked up by the provide, therefore wait for a transaction to be mined after broadcasting. See the code below for example
 
-- Transactions can fail if previous transactions are not mined and picked up by the provide, therefore wait for a transaction to be mined after broadcasting. See the code below for example
 ```javascript
 await testToken.mint(userWallet.address, 1e9).then((tx) => tx.wait(network.config.confirmations));
 
 // Or
 
-const txExecute = await interchainGovernance.execute(
-            commandIdGateway,
-            governanceChain,
-            governanceAddress,
-            payload,
-            getGasOptions(),
-        );
+const txExecute = await interchainGovernance.execute(commandIdGateway, governanceChain, governanceAddress, payload, getGasOptions());
 const receiptExecute = await txExecute.wait(network.config.confirmations);
- ```
+```
+
+-   The `changeEtherBalance` check expects one tx in a block so change in balances might need to be tested explicitly for unit tests using `changeEtherBalance`.
+
+## Deploying contracts
```

Comment:
move this earlier, right after the build/test section

Reply:
sounds good